simplesat/sat/policy.py

```diff
@@ -59,7 +59,11 @@ def _log_pretty_pkg_id(self, pkg_id):
         package = self._log_pool._id_to_package[pkg_id]
         name_ver = '{} {}'.format(package.name, package.version)
         fill = '.' if pkg_id % 2 else ''
-        return "{:{fill}<30} {}".format(name_ver, pkg_id, fill=fill)
+        try:
+            repo = package.repository_info.name
```

Comment:
I can no longer reproduce this at all. I'm not sure why it's there or if it does anything meaningful. I'm going to scrap it.